docs(v21): stesura del documento (#245) (#289)
Synergo-Unit-UniPD/Documentary-Repo: docs/PB/doc_gestionali/verb_interni/verb_int_2026_07_07/contenuto.tex

        \textbf{Nome e Cognome} & \textbf{Matricola} \\
        \midrule
        Francesco Marcon & 2110990 \\

>         Filippo Idiometri & 2035617 \\

Reviewer: filippo era presente solo che è uscito prima quindi va messo tra i presenti

Gli argomenti previsti per la riunione odierna sono:

\begin{enumerate}
    \item Sprint review e retrospective.
    \item Allineamento sulle automazioni CI/CD e metriche di codice e test.

>     \item Stesura documenti.

Reviewer: ```suggestion
    \item Applicazione delle correzioni ai documenti a seguito dell'esito RTB
```

>     \item Applicazione delle correzioni sull'AdR ed e-mail per Zucchetti.

Reviewer: ```suggestion
    \item Preparazione della comunicazione con Zucchetti.
```

    \item Pianificazione dello sprint e assegnazione ruoli.
\end{enumerate}

% --- 3. RESOCONTO E DISCUSSIONE ---

\section{Resoconto della Riunione}

\subsection{Revisione dello sprint e retrospettiva}

> Il gruppo ha effettuato la verifica dello sprint appena concluso, constatando che le attività previste sono state completate correttamente. Durante la retrospettiva è emersa la necessità di adottare una gestione più rigorosa del ciclo di lavoro, seguendo la sequenza consuntivo del periodo, retrospettiva, misure correttive con aggiornamento dei rischi, miglioramento della pianificazione futura e preventivo a finire. È stato inoltre stabilito l’avvio effettivo della Product Baseline in data 07/07, con l’obiettivo di definire le attività da inserire nel backlog e le relative priorità.

Reviewer: ```suggestion
Il gruppo ha effettuato la revisione dello sprint appena concluso, constatando che le attività previste sono state completate correttamente. Durante la retrospettiva è emersa la necessità di adottare una gestione più rigorosa del ciclo di lavoro, seguendo la sequenza consuntivo del periodo, retrospettiva, misure correttive con aggiornamento dei rischi, miglioramento della pianificazione futura e preventivo a finire. A seguito dell'esito positivo della revisione RTB, è stato stabilito l’avvio effettivo della Product Baseline in data 07/07, con l’obiettivo di definire le attività da inserire nel backlog e le relative priorità.
```

> Nel corso della discussione sono state affrontate anche le correzioni da apportare all’Analisi dei Requisiti, con la necessità di aggiornare requisiti prestazionali, tabelle di tracciamento e fonti, oltre alla numerazione dei Casi d’Uso, che verrà rivista solo al termine della stesura definitiva. È stato inoltre stabilito che i registri delle modifiche non dovranno più essere inclusi nei verbali. Il gruppo ha concordato di redigere un’e-mail per Zucchetti e di strutturare il verbale esterno includendo decisioni puntuali e informazioni utili come fonti dei Casi d’Uso. Infine, è stato evidenziato il bisogno di allinearsi sugli impegni personali dei membri per stimare correttamente il carico di lavoro individuale e pianificare la ripresa a pieno regime.

Reviewer: ```suggestion
Nel corso della discussione sono state affrontate le correzioni all’Analisi dei Requisiti emerse durante la revisione RTB, riguardanti l'aggiornamento dei requisiti prestazionali, delle tabelle di tracciamento, delle fonti e della numerazione dei Casi d’Uso, che verrà rivista al termine della stesura definitiva. È stato inoltre stabilito che i registri delle modifiche non dovranno più essere inclusi nei verbali. Il gruppo ha concordato di redigere un’e-mail per Zucchetti e di strutturare il verbale esterno includendo decisioni puntuali e informazioni utili come fonti dei Casi d’Uso. Infine, è stato evidenziato il bisogno di allinearsi sugli impegni personali dei membri per stimare correttamente il carico di lavoro individuale e pianificare la ripresa a pieno regime.
```

    \bottomrule
    \endlastfoot

    

    \textbf{VI-21.1} & Avvio effettivo della Product Baseline in data 07/07, con definizione delle attività da inserire nel backlog e delle priorità iniziali. \\

    \textbf{VI-21.2} & Adozione del ciclo di lavoro indicato dal docente: consuntivo del periodo, retrospettiva, misure correttive con aggiornamento dei rischi, miglioramento della pianificazione futura e preventivo a finire. \\

    \textbf{VI-21.3} & Strutturazione del verbale esterno per Zucchetti includendo decisioni puntuali, valutazioni tecniche e informazioni utili come fonti dei Casi d’Uso. \\

>     \textbf{VI-21.4} & Aggiornamento dell’Analisi dei Requisiti con requisiti prestazionali, tabelle di tracciamento, fonti e revisione della numerazione dei Casi d’Uso al termine della stesura definitiva. \\

Reviewer: ```suggestion
    \textbf{VI-21.4} & Aggiornamento dell’Analisi dei Requisiti con revisione dei requisiti prestazionali, delle tabelle di tracciamento e delle fonti. \\

\textbf{VI-21.5} & Revisione della numerazione dei Casi d’Uso al completamento della stesura definitiva. \\
```

>     \textbf{VI-21.5} & Rimozione dei registri delle modifiche dai verbali, come stabilito dall’esito RTB. \\

Reviewer: ```suggestion
    \textbf{VI-21.6} & Rimozione dei registri delle modifiche dai verbali, come stabilito dall’esito RTB. \\
```

>     \textbf{VI-21.6} & Avvio della sperimentazione sul setup delle automazioni CI/CD tramite branch dedicato, con analisi delle metriche di qualità del codice e dei test. \\

Reviewer: ```suggestion
    \textbf{VI-21.7} & Avvio della sperimentazione sul setup delle automazioni CI/CD tramite branch dedicato, con analisi delle metriche di qualità del codice e dei test. \\
```

>     \textbf{VI-21.7} & Allineamento sugli impegni personali dei membri per stimare correttamente il carico di lavoro individuale e pianificare la ripresa a pieno regime. \\

Reviewer: ```suggestion
```

    \bottomrule
    \endlastfoot

>     \ghissue{288} & VI-21.4 & Aggiornamento del Piano di Progetto & Francesco Marcon & 2026-07-07 \\[4pt]

Reviewer: ```suggestion
    \ghissue{288} & - & Aggiornamento del Piano di Progetto & Francesco Marcon & 2026-07-07 \\[4pt]
```

>     \ghissue{285} & VI-21.6 & Aggiornamento del Piano di Qualifica & Anton Ricardo Rupi & 2026-07-07 \\[4pt]

Reviewer: ```suggestion
    \ghissue{285} & - & Aggiornamento del Piano di Qualifica & Anton Ricardo Rupi & 2026-07-07 \\[4pt]
```

>     \ghissue{284} & - & Stesura del Verbale Interno & Roberto M. Doroftei & 2026-07-07 \\[4pt]

Reviewer: ```suggestion
    \ghissue{284} & 21.6 & Stesura del Verbale Interno & Roberto M. Doroftei & 2026-07-07 \\[4pt]
```

    \ghissue{283} & VI-21.4 & Revisione dell’Analisi dei Requisiti con aggiornamento dei use case della macroarea 2 & Filippo Idiometri & 2026-07-07 \\[4pt]

>     \ghissue{282} & VI-21.3 & Revisione dell’Analisi dei Requisiti con aggiornamento dei use case della macroarea 1 e 3 & Armando Moda Scarati & 2026-07-07 \\[4pt]

Reviewer: ```suggestion
    \ghissue{282} & VI-21.4 & Revisione dell’Analisi dei Requisiti con aggiornamento dei use case della macroarea 1 e 3 & Armando Moda Scarati & 2026-07-07 \\[4pt]
```

>     \ghissue{281} & - & Stesura verbale esterno Zucchetti & Anton Ricardo Rupi & 2026-07-07 \\[4pt]    

Reviewer: ```suggestion
    \ghissue{281} & 21.6 & Stesura verbale esterno Zucchetti & Anton Ricardo Rupi & 2026-07-07 \\[4pt]    
```

\end{longtable}

> 

Reviewer: ```suggestion
\ghissue{292} & VI-21.7 & Sperimentazione delle automazioni CI/CD e raccolta delle metriche di qualità & Francesco Marcon & 2026-07-14 \\[4pt]
```

    \bottomrule
    \endlastfoot

    \ghissue{288} & - & Aggiornamento del Piano di Progetto & Francesco Marcon & 2026-07-07 \\[4pt]

    \ghissue{285} & - & Aggiornamento del Piano di Qualifica & Anton Ricardo Rupi & 2026-07-07 \\[4pt]

    \ghissue{284} & VI-21.6 & Stesura del Verbale Interno & Roberto M. Doroftei & 2026-07-07 \\[4pt]

    \ghissue{283} & VI-21.4 & Revisione dell’Analisi dei Requisiti con aggiornamento dei use case della macroarea 2 & Filippo Idiometri & 2026-07-07 \\[4pt]

    \ghissue{282} & VI-21.4 & Revisione dell’Analisi dei Requisiti con aggiornamento dei use case della macroarea 1 e 3 & Armando Moda Scarati & 2026-07-07 \\[4pt]

    \ghissue{281} & VI-21.6 & Stesura verbale esterno Zucchetti & Anton Ricardo Rupi & 2026-07-07 \\[4pt]    

    \ghissue{292} & VI-21.7 & Sperimentazione delle automazioni CI/CD e raccolta delle metriche di qualità & Francesco Marcon & 2026-07-14 \\[4pt]

> 

Reviewer: le issue sarebbero da mettere in ordine crescente per #id
